common/bombardment_stances/hfe_stance.txt

```diff
@@ -32,6 +32,7 @@ hfe_stance_drone = {
 			or = {
 				is_country_type = hfe_drone
 				is_country_type = hfe_berserk
+				is_country_type = drone
```

Comment:
Why vanilla country type

Reply:
Flavor. HFE Ancient Mining Drones are Ancient Mining Drones. I thought it was cool to quietly establish the correlation. Also, normal Drones already bombard any colony that is founded in their system anyway.

Reply:
I see no reason why it should be

Reply:
vanilla drones have no ai and never will bombard planet

Reply:
Not true. If you colonize a planet in a system with space creatures, they WILL bombard that planet to remove its colony. I learned this when I first started playing "Stellaris", long before I added any Mods.

Reply:
I have played "Stellaris" for years. I can and have sneaked a Colony Ship past a Space Creature and colonized it. As soon as the Colony Ship lands and starts the Colony, the Space Creature moves to orbit the planet and bombard the colony out of existence.

Reply:
Before 2.0 i guess

Reply:
Anyway my statement still here: dont mess with vanilla content without reason.

Reply:
I told you the reason right there.

Reply:
What reason?
And you cant make a starbase within system with drones.
You cant colonize without starbase.
Even if you can, any type of bordbardment will kill colony.

Reply:
Okay, done